libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/TypeRep.hs

```diff
@@ -331,14 +338,15 @@ data Rep era t where
   PrevHardForkR :: Era era => Rep era (PrevGovActionId 'HardForkPurpose (EraCrypto era))
   PrevCommitteeR :: Era era => Rep era (PrevGovActionId 'CommitteePurpose (EraCrypto era))
   PrevConstitutionR :: Era era => Rep era (PrevGovActionId 'ConstitutionPurpose (EraCrypto era))
-  RatifyStateR :: Reflect era => Rep era (RatifyState era)
+  RatifyStateR :: (Reflect era, Arbitrary (PParamsHKD StrictMaybe era)) => Rep era (RatifyState era)
```

Comment:
Why is this constraint necessary now but it wasn't needed before.

I suspect that @TimSheard will be against change

Reply:
This was no longer required, after rebasing on top of master and further introducing the new datatypes correctly into the existing constrained framework. :+1: Thank you! :smile: :raised_hands: